### PR TITLE
Clear on logout

### PR DESCRIPTION
**Pull request by godu**
On logout :
- partial clear of store. We want to preserve permissions and navigation.
- clear all asyncstorage.

### CODE REVIEW

File: src/modules/version.js.flow [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// @flow
+
+declare module.exports: {|
+  brandch: string,
```
Comment (adamska27) [inline src/modules/version.js.flow]: brandch?

File: .eslintrc [hunk 2]
```diff
@@ -19,6 +19,7 @@
     }
   },
   "rules": {
+    "import/prefer-default-export": "off",
```
Comment (VincentCATILLON) [inline .eslintrc]: These rules might be in https://github.com/CoorpAcademy/eslint-plugin-coorpacademy/tree/master/config (react-native file). IMO, we should talk about it after the MVP

File: __e2e__/utils.js [hunk 3]
```diff
@@ -21,16 +21,6 @@ export const reloadApp = async (
   }
 };
 
-export const resetApp = (
```
Comment (VincentCATILLON) [inline __e2e__/utils.js]: Why did you remove this ? It's used by e2e to manage permissions

Comment (VincentCATILLON) [reply __e2e__/utils.js]: See: https://github.com/wix/Detox/blob/master/docs/APIRef.DeviceObjectAPI.md#2-set-runtime-permissions to have more informations

Comment (godu) [reply __e2e__/utils.js]: It's not used anymore.

Comment (VincentCATILLON) [reply __e2e__/utils.js]: Oh ok my bad the naming were really similar with `reloadApp` ^^
